Uplink-CLI.md

```diff
@@ -8,12 +8,12 @@ Satellite and uses a [test network](Test-network) Satellite by default.
 
 First, if you haven't already followed the preparation steps in https://github.com/storj/storj/wiki/Vanguard-Release-Setup-Instructions, please do so.
 
-Next, Download the correct binary for your operating system:
+Next, download the correct binary for your operating system: https://github.com/storj/storj/releases/latest
 
-- Mac OS: [uplink_darwin_amd64.zip](https://github.com/storj/storj/releases/download/v0.16.1/uplink_darwin_amd64.zip)
-- Linux: [uplink_linux_amd64.zip](https://github.com/storj/storj/releases/download/v0.16.1/uplink_linux_amd64.zip)
-- Raspberry Pi: [uplink_linux_arm.zip](https://github.com/storj/storj/releases/download/v0.16.1/uplink_linux_arm.zip)
-- Windows Pro: [uplink_windows_amd64.zip](https://github.com/storj/storj/releases/download/v0.16.1/uplink_windows_amd64.exe.zip)
+- Mac OS: _uplink_darwin_amd64.zip_
+- Linux: _uplink_linux_amd64.zip_
+- Raspberry Pi: _uplink_linux_arm.zip_
+- Windows: _uplink_windows_amd64.zip_
```

Comment:
You can use links like `https://github.com/storj/storj/releases/latest/download/uplink_linux_arm.zip`

Reply:
Oh, nice! I tried it, but somehow I couldn't guess the correct URL.

Reply:
https://help.github.com/en/articles/linking-to-releases